rollup.config.js

```diff
@@ -83,10 +83,8 @@ const basePlugins = ({ useBrowserResolve = false }) => [
   copy({
     targets: [
       { dest: 'dist/assets', src: './node_modules/@stream-io/stream-chat-css/dist/assets/*' },
-      { dest: 'dist/css', src: './node_modules/@stream-io/stream-chat-css/dist/css/*.css' },
-      { dest: 'dist/scss', src: './node_modules/@stream-io/stream-chat-css/dist/scss/*' },
-      { dest: 'dist/css/v2', src: './node_modules/@stream-io/stream-chat-css/dist/v2/css/*.css' },
-      { dest: 'dist/scss/v2', src: './node_modules/@stream-io/stream-chat-css/dist/v2/scss/*' },
+      { dest: 'dist/css', src: './node_modules/@stream-io/stream-chat-css/dist/v2/css/*.css' },
+      { dest: 'dist/scss', src: './node_modules/@stream-io/stream-chat-css/dist/v2/scss/*' },
```

Comment:
I think we should just remove the legacy theming from the bundle and keep the modern theming under `v2/css`. That way, this won't be a breaking change for customers who are already on v2 theming.

Reply:
I know about reducing the amount of breaking changes per such release but in this case I think leaving behind `v2` traces seems sub-optimal. Let's clean it all up while we're at it and as this is pretty noticeable breaking change, we can lead with it in the release guide. :)

Reply:
Update: after discussing this internally, we agreed to remove the `/v2` from the path.

Reply:
Update: after another discussion, we decided to keep `v2` in the path as there are even video tutorials, but also sandboxes and written tutorials using this path.

Reply:
Ref to commit: https://github.com/GetStream/stream-chat-react/pull/2394/commits/c7caaa2b88c8ac9cfd21720fc20261e21d703c27

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdTIwZWg3MmhpajYyM29kNWd3aWtqaXppc3oyZGJ2d3VzYTVrY2M1MyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/tgpFkX8JkxmnvXhiWc/giphy.gif)